file.go

```diff
@@ -39,8 +38,7 @@ type File struct {
 	delta   []*Undo // [private]
 	epsilon []*Undo // [private]
 	elog    Elog
-	name    string
-	info    os.FileInfo
+	details *file.DiskDetails
```

Comment:
It makes more sense to put this in `ObservableEditableBuffer`. Presumably you'll do this eventually.

Reply:
Yes, that's on the list